contrib/format-excel/src/main/java/org/apache/drill/exec/store/excel/ExcelFormatConfig.java

```diff
@@ -0,0 +1,117 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.drill.exec.store.excel;
+
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.fasterxml.jackson.annotation.JsonTypeName;
+import org.apache.drill.shaded.guava.com.google.common.base.Objects;
+import org.apache.drill.shaded.guava.com.google.common.collect.ImmutableList;
+import org.apache.drill.common.logical.FormatPluginConfig;
+import org.apache.drill.exec.store.excel.ExcelBatchReader.ExcelReaderConfig;
+
+import java.util.Arrays;
+import java.util.List;
+
+@JsonTypeName(ExcelFormatPlugin.DEFAULT_NAME)
+@JsonInclude(JsonInclude.Include.NON_DEFAULT)
+public class ExcelFormatConfig implements FormatPluginConfig {
+
+  private static final List<String> DEFAULT_EXTS = ImmutableList.of("xlsx");
+
+  // This is the theoretical maximum number of rows in an Excel spreadsheet
+  private final int MAX_ROWS = 1048576;
+
+  public List<String> extensions;
+
+  public int headerRow;
+
+  public int lastRow = MAX_ROWS;
+
+  public int firstColumn;
+
+  public int lastColumn;
+
+  public boolean allTextMode;
+
+  public String sheetName = "";
+
+
+  public ExcelFormatConfig() {
```

Comment:
There are not other constructors thus no need to add default one...

Reply:
Fixed